### ISSUE TITLE
Screen reader does not announce the visual label for the placeholder.

### ISSUE THREAD

**Issue by mohitrock1** [ckeditor/ckeditor4#3945]
**User Experience:**
If screen reader does not provide proper information about the control present on the screen then it might gets confuse the user about the functionality about the control.

**Note: User credentials should NOT be included in the bug.**

**Repro Steps:**
1. Launch URL: https://codepen.io/flesler/pen/AEIFc
2. Navigate to enter text here.
3. Verify the issue.

![image](https://user-images.githubusercontent.com/29257206/77290249-ce320180-6d01-11ea-99d6-b32fa5935762.png)

**Actual Results:**
Screen reader does not announce the visual label 'Enter text here ...' for the edit box present on the screen.

**Expected Result:**
Screen reader should announce the visual label 'Enter text here ...' for the edit box present on the screen.

**Note:**
Same issue repro with Chrome+JAWS and Narrator+Edge.

**Reference Links:**
Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**Comment by f1ames** [comment]
Hello @mohitrock1, is this issue related to CKEditor 4 in any way? I don't see it in the codepen demo :thinking:

**Comment by Dumluregn** [comment]
Hi, It's been a while since we last heard from you. We're closing this issue for now. Still, feel free provide us requested feedback, so that we can reopen this issue.
